bedrock/firefox/templates/firefox/tracking-protection-tour.html

```diff
@@ -136,3 +136,6 @@
 {% block site_js %}
   {% javascript 'tracking-protection-tour' %}
 {% endblock %}
+
+{#- Ensure that this page never gets the New Relic JS -#}
+{% block newrelic %}{% endblock %}
```

Comment:
Could actually move this up to the top of the template next to where we already block out GA?